x-pack/plugins/infra/public/pages/metrics/inventory_view/components/layout.tsx

```diff
@@ -172,17 +181,24 @@ export const Layout = React.memo(
               </EuiFlexGroup>
             </TopActionContainer>
             <EuiFlexItem grow={false}>
-              <TryItButton
-                data-test-subj="inventory-hostsView-link"
-                label={i18n.translate('xpack.infra.layout.hostsLandingPageLink', {
-                  defaultMessage: 'Introducing a new Hosts analysis experience',
-                })}
-                link={{
-                  app: 'metrics',
-                  pathname: '/hosts',
-                }}
-                experimental
-              />
+              {!hostsLinkClickedRef?.current && (
+                <TryItButton
+                  data-test-subj="inventory-hostsView-link"
+                  label={i18n.translate('xpack.infra.layout.hostsLandingPageLink', {
+                    defaultMessage: 'Introducing a new Hosts analysis experience',
+                  })}
+                  link={{
+                    app: 'metrics',
+                    pathname: '/hosts',
+                  }}
+                  experimental
+                  onClick={() => {
+                    if (!hostsLinkClickedRef.current) {
```

Comment:
**nit**: is this conditional required? since the button won't appear anymore on following visits, the user won't event have the chance to reach this handler as the button is not on the screen.